.circleci/config.yml

```diff
@@ -157,14 +157,22 @@ commands:
       - run:
           name: Check for any unrecorded changes in our dependency trees
           command: |
-            pip3 install requests
+            pip3 install --require-hashes -r ./tools/requirements.txt
```

Comment:
hashes are cool, we should verify them on install wherever possible

Reply:
Oh nice we should do the same for our regular CI